index.html

```diff
@@ -546,6 +546,23 @@ <h3 data-tests="wakelock-onrelease.https.html">
           {{WakeLockSentinel/onrelease}} event handler.
         </aside>
       </section>
+      <section>
+        <h3>
+          Garbage collection
+        </h3>
+        <p>
+          While a {{WakeLockSentinel}} object has one or more event listeners
+          registered for "release", there MUST be a strong reference from the
```

Comment:
In the Chromium implementation we also check that the lock has not been released, as a released lock will never fire the "release" event again.